docs/components/secnav/index.html

```diff
@@ -28,14 +28,14 @@ <h3>
       </h3>
       <br />
     </header>
-
+  
     <div id="vue-secNavDemo">
       <nav id="hxTopNav">
         <a class="hxTopNavLogo" href="#">
           <img src="images/helix-logo.svg" alt="Brand Logo" />
         </a>
         <div class="hxTopNavMenu hxTopNavOptionMenu">
-          <hx-disclosure aria-controls="topnav-menu-options">
+          <hx-disclosure aria-controls="topnav-menu-options" aria-expanded="true">
```

Comment:
Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

Reply:
Changes have been done, please review.